ONSdigital/dp-frontend-router
# Added meta tag for service name

### What

Service name added to error pages in the meta tag within the head of the page

### How to review

1. Checkout this branch and load the router locally
2. Launch the web page and obtain an error page (easiest to accomplish when not running other services).
3. Open the page inspector (Developer console)
4. The following should be present <meta name="ons:service" content="dp-frontend-renderer">

### Who can review

Anyone with access to the router and is able to inspect the page elements.
